src/server/scripts/Northrend/CrusadersColiseum/TrialOfTheChampion/instance_trial_of_the_champion.cpp

```diff
@@ -74,10 +75,11 @@ class instance_trial_of_the_champion : public InstanceMapScript
             memset(&NPC_GrandChampionMinionsGUID, 0, sizeof(NPC_GrandChampionMinionsGUID));
             memset(&NPC_ArgentSoldierGUID, 0, sizeof(NPC_ArgentSoldierGUID));
             NPC_ArgentChampionGUID = 0;
-            NPC_MemoryEntry = 0;
+1            NPC_MemoryEntry = 0;
```

Comment:
1

Reply:
what?

the error in travis makes no sense though 

/home/travis/build/azerothcore/azerothcore-wotlk/src/server/scripts/Northrend/CrusadersColiseum/TrialOfTheChampion/instance_trial_of_the_champion.cpp:78:2: fatal error: 
      expected ';' after expression
1            NPC_MemoryEntry = 0;